packages/bson-transpilers/test/yaml/driver-syntax.yaml

```diff
@@ -434,6 +461,27 @@ tests:
                     }).skip(5).limit(10).maxTimeMS(100).collation({
                       'locale': 'zh'
                     });"
+                php: |-
+                    // Requires the MongoDB PHP Driver
+                    // https://www.mongodb.com/docs/drivers/php/
+
+                    $client = new \MongoDB\Client('mongodb://localhost:27017');
+                    $db = $client->selectDatabase('db');
```

Comment:
`$db` isn't used anywhere else. Either replace with this:

```
$collection = $client->selectCollection('db', 'collection');
```

...and use `$collection->find()` later, or simply avoid creating extra variables and use `$client->db->collection->find()`. I'm partial to using `selectCollection()`, as that's what we do in our own documentation.

Reply:
Fixed.

Reply:
Can you point me to the fix? I'm wondering if this line should now be updated in the PR.